terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -455,19 +402,19 @@ module "s3_glue_job_bucket" {
   )
 }
 
-# S3 Landing
-module "s3_landing_bucket" {
+# S3 Raw Archive
+module "s3_raw_archive_bucket" {
```

Comment:
Creates the raw_archive bucket where parquet files will be archived/transferred after processing